src/vscode/superbol-vscode-platform/superbol_languageclient.ml

```diff
@@ -12,13 +12,18 @@
 (*                                                                        *)
 (**************************************************************************)
 
-let serverOptions () =
+let serverOptions superbol_path =
```

Comment:
Please could you label this argument? Otherwise it's hard to guess what is expected by `Superbol_languageclient.serverOptions`…

Reply:
Will do, thought I did already.